.github/workflows/dockerhub-pr-with-parameters.yml

```diff
@@ -0,0 +1,101 @@
+on: issue_comment
+name: DockerHub (PR) with parameters
+
+jobs:
+  check_comments:
+    name: Check comments for /build_pr
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for Command
+        id: command
+        uses: xt0rted/slash-command-action@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          command: build_pr
+          reaction: "true"
+          reaction-type: "eyes"
+          allow-edits: "false"
+          permission-level: maintain
+      - name: Act on the command
+        run: echo "The command was '${{ steps.command.outputs.command-name }}' with arguments '${{ steps.command.outputs.command-arguments }}'"
+      # Build container
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+      - run: git rev-parse --abbrev-ref HEAD
+      - run: git rev-parse --verify HEAD
+      - name: Clone branch of the PR
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ steps.pr_data.outputs.branch }}
+      - name: build-tag-push-pr
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          dockerfile: docker/Dockerfile
+          cache_froms: buildpack-deps:stable-curl
+          repository: xaynetwork/xaynet
+          tags: pr-${{ steps.pr_data.outputs.branch }}
+          push: true
+          build_args: '${{ steps.command.outputs.command-arguments }}'
+      # k8s deploy
+      - name: build-tag-push
```

Comment:
ahh just discovered a problem
once we restart the coordinator pod, k8s tries to pull the docker image with the tag `development` but in our case it is the name of the branch. Not sure how we can make this configurable. I think we need to think about it.

Reply:
This action seems to solve this necessity 
```
      - name: Find and Replace
        uses: jacobtomlinson/gha-find-replace@master
```

Reply:
Nice!
can you please remove the `build-tag-push-pr` (43-L53) step because we have this step twice

Reply:
Done